Contributing.md

```diff
@@ -35,3 +35,8 @@ This project uses [Discussions](https://github.com/MetroCS/redistricting/discuss
    $ ant all 
    $ cd build
 ```
+#### Ubuntu (22.04) & Bash
+If when building you're missing the jar files and junit tests aren't running, then you should 
+download ant through snapd using the following commands(once youve installed [snapd](https://snapcraft.io/docs/installing-snap-on-ubuntu)).
+#### MAC OS 
+If java tests are failing to complie, try downloading ant through [homebrew](https://brew.sh/)
```

Comment:
This issue is now specifically in regards to build failing because junit.test.launcher isn't in the classpath. In my revision I added some possible reasons one might get the error "junit.test.launcher" not in classpath. Although this phrasing of the error implies that its a possible error, given that all of my teammates experienced this problem and dealt with this frustration, and given that this error can happen for various reasons, I think its a needed 'tip' or 'disclaimer'.